pkg/kafka/topic.go

```diff
@@ -0,0 +1,31 @@
+package kafka
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/justtrackio/gosoline/pkg/cfg"
+)
+
+func NormalizeTopicName(name string) string {
+	return strings.ReplaceAll(name, "_", "-")
+}
+
+// FQTopicName returns fully-qualified topic name.
+func FQTopicName(appId cfg.AppId, topic string) string {
+	return NormalizeTopicName(namingStrategy(appId, topic))
+}
+
+// DLTopicName returns dead letter topic name.
+func DLTopicName(topic, groupID string) string {
+	return NormalizeTopicName(
+		fmt.Sprintf(
+			"%s.%s.dl",
+			topic,
+			groupID,
+		))
+}
+
+func namingStrategy(appId cfg.AppId, topic string) string {
```

Comment:
Is there any way to overwrite the naming strategy?  
That would be a good option to have.

Reply:
done.